docs/assignment1.md

```diff
@@ -1,62 +1,52 @@
 
-As a preamble to the table, you can discuss the main line of reasoning you applied for defining the functional features provided in the table.
+Existing snippet manager applications for inspiration:
+1. Gisto: https://www.gistoapp.com/features
+2. Github Gists: https://developer.github.com/v3/gists/.
 
-| ID  | Short name  | Description  |
-|---|---|---|
-| F1  | Tags | Code snippets can be tagged via freely-defined labels called tags  |
-| F2  | Commands  | The player can control the main character by issuing command-line commands following this syntax: `command-name [target-objects]*`. The available `command-names` are the following: <br/> - move: ... <br/> - use: ... <br/> - inspect: ... <br/> |
-| F3  | Movements  | The main character shall move freely in the environment according  |
-| F4  | ... | ... |
+## Features:
 
-### Quality requirements
-Author(s): `name of the team member(s) responsible for this section`
+Author(s):Serghei Mihailov, Bjorn Keyser, Yael Goede, Milos Delgorge
 
-As a preamble to the table, you can discuss the main line of reasoning you followed for coming up with the quality requirements listed below.
+| **ID** | **Short name** | **Description** |
+| --- | --- | --- |
+| F1 | API to Create/Read/Update/Delete snippets | Users should be able to easily access edit, create and delete snippets by calling the tool from the CLI using arguments. |
+| F2 | Snippet editor | Users should be able to edit and save snippets via a basic Vim-like CLI editor with visual/insert/replace modes. |
+| F3 | Tags | Users should be able to add tags to snippets by which will make finding snippets easier. This should include language and whether the snippet is "starred". |
+| F5 | Search/Filter | Users can search for their snippets by content, tags, description, title, language. Like snippo -s "language:java tags:tag1,tag2 searchterm". |
+| F6 | Configurable syntax highlighting | Users should have their code for a specific language with highlighting of syntax that is configurable via a file (that they create or that has been provided) that maps keywords to colors. |
+| F7 | Readme/help for the tool | Users should be able to view a manual explaining the basic commands and how to use the tool. |
+| F8 (optional) | Authentication | Users should be able to create and use accounts where their personal protected snippets are stored. |
```

Comment:
[SEVERE] What do you mean here? Will you have a database with user account information, will the information be stored locally?